src/cookieconsent.js

```diff
@@ -132,6 +134,24 @@
       // The code "typeof obj === 'object' && obj !== null" allows Array objects
       return typeof obj === 'object' && obj !== null && obj.constructor == Object;
     },
+
+    loadStylesheet: function (stylesheet, complete) {
+      var link = document.createElement("link");
```

Comment:
I think we're probably best to remove it then. Maybe we could have a section of the configurator for common use cases with example code.